public/js/app/components/statistics/Chart.jsx

```diff
@@ -10,8 +10,16 @@ const colors = {
   grey: '#65656C',
   pink: '#D02F80',
   red: '#B52C17',
+  yellow: '#fab919',
+}
+const schoolsColors = {
+  ABE: colors.red,
+  CBH: colors.grey,
+  EECS: colors.green,
+  ITM: colors.blue,
+  SCI: colors.pink,
+  'All schools': colors.yellow,
```

Comment:
`All schools`  better to be replaced with `allSchools`

Reply:
done